metricflow-semantics/tests_metricflow_semantics/snapshots/test_time_adjuster.py/str/test_start_and_end_periods__start_and_end_of_period_results.txt

```diff
@@ -0,0 +1,3652 @@
+                     Date                 Period Grain         Period Start         Period End
```

Comment:
This looks like a leftover from an earlier version of the test. Remove?

Reply:
Removed.